File: solutions/beecrowd/1113/1113.c
#include <stdint.h>
#include <stdio.h>

int main() {
    int16_t a, b;

    while (scanf("%d %d", &a, &b) && a != b) {
        if (a < b) {
            printf("Crescente\n");
        } else {
            printf("Decrescente\n");
        }
    }

    return 0;
}

File: solutions/beecrowd/1113/1113.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

while line := input().strip():
    a, b = map(int, line.split())
    if a == b:
        break

    if a < b:
        output('Crescente\n')
    else:
        output('Decrescente\n')
